### PR TITLE
Update/0.11.4

### PR DESCRIPTION
**Pull request by ryanskeith**
mizani 11.4

**Destination channel:** {defaults}

### Links

- [Upstream repository](https://github.com/has2k1/mizani/blob/v0.11.4/pyproject.toml)
- [Upstream changelog/diff](https://github.com/has2k1/mizani/compare/v0.9.2..v0.11.4)

### Explanation of changes:
- Version and Hash updates
- Changed imports to test for as they were deprecated
- Changed test commands to be simpler

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,65 +1,56 @@
...
+    - pyproject.toml
   commands:
-    # The pip version can reported incorrectly if setuptools_scm isn't available
-    # at build time
-    - pip list | grep -i {{ name }} | grep {{ version }}
-    - pip check
-    - echo $PWD
-    - ls -larth
-    - pytest
+    - pip check  # [not win]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: Could you please add a comment on why there is skipping for `win`?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,65 +1,56 @@
...
+    - pytest -vv
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:43]: The test failed on `win64` for **py311**:

```
>           raise ValueError(

                "Cannot get a null value for type: {}".format(type(x[0]))

E           ValueError: Cannot get a null value for type: <class 'numpy.int32'>

..\_test_env\Lib\site-packages\mizani\utils.py:387: ValueError
...

FAILED tests/test_bounds.py::test_censor - ValueError: Cannot get a null value
```

Is this an upstream issue or other `numpy` incompatibility?

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,65 +1,56 @@
...
     - pytest
     - pytest-cov
-  imports:
-    - mizani
-    - mizani.external
-    - mizani.tests
   source_files:
-    - '*'
+    - "tests"
+    - pyproject.toml
```
Comment (danpetry) [inline recipe/meta.yaml:40]: out of interest is this required to find the tests, or..?

**Review by danpetry — APPROVED** [review]
Approved based on addressing Serhii's comments, in particular the test failure (fix or skip with reasoning)

**Pull request merged**